salt/elife-dashboard/config/opt-ubr-alt-app.cfg

```diff
@@ -0,0 +1,16 @@
+[general]
+working_dir=/tmp
+descriptor_dir=/etc/ubr2/
+
+[mysql]
```

Comment:
can this be removed since the data are clearly fake? That's a UBR change probably

Reply:
can't actually, I tried it. UBR requires it to be there and I wanted my changes to be minimal